.gitignore

```diff
@@ -5,6 +5,8 @@
 *.so
 *.dylib
 *.c1z
+private/
+local_only/
```

Comment:
you can exclude these locally in your git, I don't think this should be committed, there's a local to you only way to ignore things that you want to ignore which you can see how to do via this question I asked on SO in 2009 https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally